### PR TITLE
fix: minor nit

### PR DESCRIPTION
**Pull request by ebati**
I send multiple small commits in single PR, I hope that is OK.

The biggest change is the dependencies (`go.mod` and `go.sum` files), that is because I removed the blank imports which removed the dependency to lima module and I also updated dependencies to the latest versions.

### CODE REVIEW

File: cmd/start.go [hunk 1]
```diff
@@ -121,7 +119,7 @@ $ buildctl --addr unix://$(pwd)/buildkitd.sock build ...
 		if flagUnix != "" {
 			sshOptions = append(sshOptions, "-nNT")
 			rootlessBuildkitdSockPath := fmt.Sprintf("/run/user/%s/buildkit/buildkitd.sock", strings.TrimSpace(string(uid)))
-			println(rootlessBuildkitdSockPath)
```
Comment (developer-guy) [inline cmd/start.go:124]: you can remove it, we don't need to print the socket path.

**Review by developer-guy — CHANGES_REQUESTED** [review]
just a few minor changes requested, otherwise, LGTM, thanks for the PR @ebati

**Comment by Dentrax** [comment]
LGTM! Don't forget to squash your commits. :)

**Comment by developer-guy** [comment]
thanks for the PR @ebati

**Pull request merged**